plugins/auth-backend/package.json

```diff
@@ -24,12 +24,16 @@
     "helmet": "^3.22.0",
     "morgan": "^1.10.0",
     "winston": "^3.2.1",
-    "yn": "^4.0.0"
+    "yn": "^4.0.0",
+    "passport": "0.4.1",
+    "passport-google-oauth20": "2.0.0"
   },
   "devDependencies": {
     "@backstage/cli": "^0.1.1-alpha.5",
     "jest-fetch-mock": "^3.0.3",
-    "tsc-watch": "^4.2.3"
+    "tsc-watch": "^4.2.3",
+    "@types/passport": "1.0.3",
+    "@types/passport-google-oauth20": "2.0.3"
```

Comment:
@types should be in dependencies

Reply:
I'll move it. I am confused about the types being deps vs. devDeps

Reply:
https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies